src/main/java/core/basesyntax/Information.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Information {
```

Comment:
Don't use verbs, better to call it `Drawable` and have method `draw` instead of `printInformation`